start.sh

```diff
@@ -146,6 +146,12 @@ else
   export PLATFORM="-$ARCH"
 fi
 
+# If using Msys2 on Windows, replace msys with windows
+UNAME_O=$(uname -o);
```

Comment:
Just as an FYI, on macOS at least, `-o` isn't an option for uname and it spits out an error
```
johns-mbp-3:codewind johncollier$ uname -o
uname: illegal option -- o
usage: uname [-amnprsv]
```

This won't prevent the script from working on macOS, but we may not want it to display that error.

Reply:
Fixed: I'm now just using the previous `uname` val.